benchmarks/benchmark_dht.py

```diff
@@ -5,6 +5,7 @@
 from tqdm import trange
 
 import hivemind
+from hivemind.dht import DHT_DEFAULT_NUM_WORKERS
```

Comment:
I'd suggest to revert changes in `benchmark_dht.py` since we already have one way to specify `max_workers` (via the env variable).